packages/mdc-auto-init/index.js

```diff
@@ -41,14 +42,18 @@ export default function mdcAutoInit(root = document, warn = CONSOLE_WARN) {
     }
 
     // TODO: Should we make an eslint rule for an attachTo() static method?
-    const component = Ctor.attachTo(node);
+    component = Ctor.attachTo(node);
     Object.defineProperty(node, ctorName, {
       value: component,
       writable: false,
       enumerable: false,
       configurable: true,
     });
   }
+
+  if (component !== null) {
```

Comment:
Probably I got this wrong. 

If each component emits its own `init:end` event when done, how would you know when all of components are initialzied? Or maybe we can use combination of both? Each component can emit own event when initialized, like `ComponentNameInit:End` and then 'MDCAutoInit:End' when all of them are initialized?